test/run_all.sh

```diff
@@ -4,76 +4,35 @@ set -e #-x
 
 WORK_DIR=test
 NGINX_PREFIX=$PWD/test-root
+LOG_PATH=$NGINX_PREFIX/logs
```

Comment:
I would also this path to README or show log file paths on the terminal during testing. The latter looks better.

Otherwise one will need to dive into the code to find logs location.

Reply:
Done.

Reply:
The log directory will be shown only after successful execution of all tests. If any of the tests fails, it will not be shown. I would show a log file path before each test run or show a directory with logs before executing a first test.

Reply:
I prefer to show a log file path before each test.

Reply:
I see that you do it before all tests, but it is okay anyway.